user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

```diff
@@ -0,0 +1,160 @@
+
+Administrators are the users of an admin panel of a Strapi application. Administrator accounts and roles are managed with the Role-Based Access Control (RBAC) feature. It is available in the *Administration panel* category of the Settings interface, accessible from *General > Settings* in the main navigation of the admin panel.
+
+The *Administration panel* category is divided into 2 sub-categories: *Roles* and *Users* (see [Managing administrators](managing-administrators.md)).
+
+![Administrator roles interface](../assets/users-permissions/administrator_roles.png)
+
+The *Roles* sub-category of *Administration panel* displays all created roles for the administrators of your Strapi application.
+
+From this interface, it is possible to:
+
+- create a new administrator role (see [Creating a new role](#creating-a-new-role)),
+- delete an administrator role (see [Deleting a role](#deleting-a-role)),
+- or access information regarding an administrator role, and edit it (see [Editing a role](#editing-a-role)).
+
+By default, 3 administrator roles are defined for any Strapi application:
+
+- Author: to be able to create, manage and publish their own content.
+- Editor: to be able to create content, and manage and publish any content.
+- Super Admin: to be able to access all features and settings. This is the role attributed by default to the first administrator at the creation of the Strapi application.
+
+::: warning ATTENTION
+If you use your Strapi application with the Community Edition (see [Pricing and Plans](https://strapi.io/pricing)), your use of the RBAC feature will be limited. Only the 3 default roles are indeed available, as you cannot create more roles and cannot delete the default ones. It is however possible to edit them, but to an extent:
+
+- Configuring permissions in detail is only available for the Enterprise Edition: with the Community Edition, if you have access to a content type, it is automatically a full access with all permissions.
+- You can only configure permissions for the content types, but not for the plugins and settings of the Strapi application.
+- Custom conditions defined for a specific permission are also only available for the Enterprise Edition.
+
+:::
+
+## Creating a new role
+
+On the top right side of the *Administration panel > Roles* interface, an **Add new role** button is displayed. It allows to create a new role for administrators of your Strapi application.
+
+To create a new role, click on the **Add new role** button. 
+Clicking on the **Add new role** button will redirect you to the roles edition interface, where you will be able to edit the role's details and configure its permissions (see [Editing a role](#editing-role-s-details)).
+
+::: tip 💡 TIP
+In the *Roles* interface, from the table, you can click on the duplicate button ![icon duplicate](../assets/users-permissions/icon_duplicate.png) to create a new role by duplicating an existing one.
+:::
+
+## Deleting a role
```

Comment:
Should we add a EE label?

Reply:
Yes, see: https://github.com/strapi/documentation/pull/118#discussion_r585079411